catalog/core/catalog-core-commands/src/test/groovy/org/codice/ddf/commands/catalog/ExportCommandSpec.groovy

```diff
@@ -170,7 +170,7 @@ class ExportCommandSpec extends Specification {
         setup:
         InputStream keyboardInput = new ByteArrayInputStream("n\r".getBytes('utf-8'))
         def session = Mock(Session) {
-            getKeyboard() >> keyboardInput
+            readLine(_ as String, null) >> keyboardInput
```

Comment:
Even though this test may not fail I don't believe it is correct. This is basically making the readLine method on the mock session return an InputStream. I think groovy doesn't barf since it is so dynamic. But readLine should return a `String` probably instead of the `InputStream` that the *getKeyboard* method was returning previously.. does that sound right?

Reply:
Fixed. Good find:+1: